docs/using/event-types.md

```diff
@@ -47,81 +47,98 @@ have a more specific definition :)
 
-By default, Nakadi uses the forward mode, which is further explained
-below, together with the two other modes: compatible and none.
+Nakadi supports different compatibility modes for event types used to
+control valid schema changes and to ensure schema consistency of
+published event data. Compatibility modes define restrictions for
+event schema evolution, i.e. a set cascade of allowed schema changes
+and thereby different compatibility guarantees for event consumers.
 
-#### Forward
+The default compatibility mode forward compatible, but full
+compatibility usage is definitely encouraged and default mode will
+change in near future.
 
-This mode has been designed to allow event type owners to change their
-schemas without breaking existing consumers.
+#### Fully compatible
 
-It's called forward because using older schemas, it's safe to read
-events generated by newer schemas. On the other hand, it's not
-backward compatible, in the sense that trying to read an older event
-using a newer schema is not safe.
+The `compatible` compatibility mode is the safest mode. It's both
+forward compatible and backward compatible. It means that:
 
-For example, if the consumer has been developed with schema version
-`1.5.0` in mind, reading events from `1.5.0` on is safe, for example,
-`1.6.0`. But reading event generated by older schemas, like `1.4.0`,
-is not safe.
+1. Consumers using older schemas can safely read events validated by
+   newer schemas.
+2. Consumers using newer schemas can safely read events validated by
+   older schemas.
+
+It guarantees high data quality which is crucial for a variety of
+applications. At Zalando, it's required to use this compatibility mode
+to have data processed by business intelligence and long term event
+storage by the data lake.
 
 Supported changes:
 
-1. changes to meta attributes: titles, descriptions and examples.
-2. addition of new attributes, both optional and required one.
-3. marking attributes as required.
+1. changes to meta attributes: titles and descriptions.
+2. addition of new optional attributes.
 
-Consumers reading events from forward compatible event types SHOULD
-ignore event attributes not declared in the event type schema.
+The following json-schema attributes are not unsupported:
 
-This mode is default because it solves the most common requirement:
-guarantee the stability of deployed consumers applications when the
-schema changes. It's crucial for the robustness of event driven
-applications.
+- additionalProperties
+- additionalItems
+- not
+- patternProperties
 
-#### Compatible
+Removing the support for these attributes is necessary to avoid the
+introduction of incompatible changes.
 
-This is the most strict mode. It's both forward compatible and
-backward compatible. In practical terms:
+Under this compatibility mode it's necessary to fully specify events
+properties in order for validation to succeed. I.e. events with
```

Comment:
In fact I think it would be better to change this: `succeed. I.e. events with properties not declared in the schema will be rejected` -> `succeed; events containing properties that are not declared in the schema will be rejected.`